File: dayTwo/Java/After Break/Test/TestQuestionOne.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class TestQuestionOne{

    @Test
    void testThatWhenIEnterFatherAgeAndSonAgeIGetTheYearsWhenFatherWillBeTwiceAsOld(){
        int fatherAge = 3;
        int sonAge = 15;
        
        int expectedAnswer = QuestionOne.fatherAgeDifference(fatherAge,sonAge);
        int actualAnswer = 0;
        
        assertEquals(expectedAnswer, actualAnswer);    
    }
}

File: dayTwo/python/After Break/test_question_one.py
import unittest

from question_one import father_age_difference

class TestFatherAgeDifference(unittest.TestCase):
    
    def test_thatWhen_iEnterFatherAge_andSonAge_IgetTheYear_theFatherWill_beTwiceTheSonsAGe(self):
        expectedAnswer = father_age_difference(30,15)
        
        actualAnswer = 0
        
        self.assertEqual(expectedAnswer,actualAnswer)
        
